v2.14.5

laurent-laporte-pro opened the pull request:
v2.14.5 on master

MartinBelthle committed:
fix(command): improve INI reader to support API PUT `/v1/studies/{uuid}/raw` (#1461)

MartinBelthle committed:
fix(web): change API response_model to avoid ValidationError (#1526)

laurent-laporte-pro committed:
build: prepare next bugfix release v2.14.5 (unreleased)

ghost commented:
### Build System

* prepare next bugfix release v2.14.5 (unreleased) ([146d11a](https://github.com/AntaresSimulatorTeam/AntaREST/commit/146d11a660d44a87dfaf08a54425065689328da7))
* new bugfix release v2.14.5 (2023-08-11) ([b090f44](https://github.com/AntaresSimulatorTeam/AntaREST/commit/b090f44a898d146acdf011161c1cc3ce8a7d0406))


### Bug Fixes

*  **web:** change API response_model to avoid ValidationError (#1526) ([b0e48d1](https://github.com/AntaresSimulatorTeam/AntaREST/commit/b0e48d1bd31463cb6ce5e9aefeff761c016d0b35))
*  **xpansion:** correct Xpansion settings field types ([3e481b9](https://github.com/AntaresSimulatorTeam/AntaREST/commit/3e481b9c8866ecc3dc42e351552e1ded036f62ad))
*  **variant:** correct implementation of `VariantCommandsExtractor.diff` method ([c534785](https://github.com/AntaresSimulatorTeam/AntaREST/commit/c5347851da867a19b990e05c6516bedc7508c8ce))
*  **api:** add missing `use_leeway` field and validation rules (#1650) ([27e46e5](https://github.com/AntaresSimulatorTeam/AntaREST/commit/27e46e5bda77aed65c84e82931d426b4b69a43bd))
*  **export:** ZIP outputs are now uncompressed before export (#1656) ([cba6261](https://github.com/AntaresSimulatorTeam/AntaREST/commit/cba62613e19712240f74f417854e95bd588ba95d))
*  **log-parser:** simplify parsing and make it accurate (#1682) ([2442674](https://github.com/AntaresSimulatorTeam/AntaREST/commit/24426749e9b6100eb3ab4b7159f615444242b95a))
*  **table-mode:** fixes the reading of UI information in the case where the study has only one area  (#1674) ([55c4181](https://github.com/AntaresSimulatorTeam/AntaREST/commit/55c4181b64959c5e191fed2256437fc95787199f))
*  **table-mode:** issue to read area information in the case where the study has only one area (#1690) ([87d9617](https://github.com/AntaresSimulatorTeam/AntaREST/commit/87d961703cebdc037671fe73988903eb14dd9547))
*  **command:** improve INI reader to support API PUT `/v1/studies/{uuid}/raw` (#1461) ([9e5cf25](https://github.com/AntaresSimulatorTeam/AntaREST/commit/9e5cf25b2f69890016ea36f3be0e9ac03c7695b6))
*  **variant:** fixed time series deletion of renewable clusters (#1693) ([4ba1b17](https://github.com/AntaresSimulatorTeam/AntaREST/commit/4ba1b17dd3c1b8ea62a5a02f39d15e94a4b9a331))
*  **launcher:** fixing launcher versions display and API endpoint for solver versions ([410afc2](https://github.com/AntaresSimulatorTeam/AntaREST/commit/410afc2e4ecbb296878985839ee27f84bc70d9d8))
*  **launcher:** set the default number of cores to 22 (instead of 12) (#1695) ([2c89799](https://github.com/AntaresSimulatorTeam/AntaREST/commit/2c8979916d46a0ed46a67bc75ac9a2e365e3f164))
*  **launcher:** correct endpoint for retrieving solver version list (#1672) ([a76f3a9](https://github.com/AntaresSimulatorTeam/AntaREST/commit/a76f3a9f01df0225d7fb54b20ba3ff599d749138))


### Continuous Integration

* upgrade mypy to v1.4.1 and Black to v23.7.0 for improved typing and formatting (#1685) ([7cff8c5](https://github.com/AntaresSimulatorTeam/AntaREST/commit/7cff8c56c38728a1b29eae0221bcc8226e9ca80c))


### Features

*  **ui-i18n:** add missing adequacy patch translations (#1680) ([8a06461](https://github.com/AntaresSimulatorTeam/AntaREST/commit/8a06461f4118227b94be7f587d37ea2430c70505))
*  **ui:** remove patch number in version list when it's equal to 0 (#1698) ([1bc0156](https://github.com/AntaresSimulatorTeam/AntaREST/commit/1bc0156c3e08e321e9ccc396b95cadeabf1c1fc7))


### Tests

* enhanced integration tests: refactored fixtures and resources ([70af9b1](https://github.com/AntaresSimulatorTeam/AntaREST/commit/70af9b167bf54d534696da8b781edda56ccee788))


### Contributors

<a href="https://github.com/laurent-laporte-pro">laurent-laporte-pro</a>, <a href="https://github.com/MartinBelthle">MartinBelthle</a>, <a href="https://github.com/hdinia">hdinia</a>, <a href="https://github.com/skamril">skamril</a>

<details>
<summary>Commit-Lint commands</summary>
<br />

You can trigger Commit-Lint actions by commenting on this PR:
- `@Commit-Lint merge patch` will merge dependabot PR on "patch" versions (X.X.Y - Y change)
- `@Commit-Lint merge minor` will merge dependabot PR on "minor" versions (X.Y.Y - Y change)
- `@Commit-Lint merge major` will merge dependabot PR on "major" versions (Y.Y.Y - Y change)
- `@Commit-Lint merge disable` will desactivate merge dependabot PR
- `@Commit-Lint review` will approve dependabot PR
- `@Commit-Lint stop review` will stop approve dependabot PR



</details>

MartinBelthle committed:
fix(export): ZIP outputs are now uncompressed before export (#1656)

hdinia committed:
fix(api): add missing `use_leeway` field and validation rules (#1650)

MartinBelthle committed:
fix(launcher): fixing launcher versions display and API endpoint for solver versions

hdinia committed:
feat(ui-i18n): add missing adequacy patch translations (#1680)

laurent-laporte-pro committed:
fix(xpansion): correct Xpansion settings field types

laurent-laporte-pro committed:
fix(variant): correct implementation of `VariantCommandsExtractor.diff` method

Take into account `CREATE_RENEWABLES_CLUSTER` and `REMOVE_RENEWABLES_CLUSTER`.

laurent-laporte-pro committed:
ci: upgrade mypy to v1.4.1 and Black to v23.7.0 for improved typing and formatting (#1685)

MartinBelthle committed:
fix(log-parser): simplify parsing and make it accurate (#1682)

Co-authored-by: Laurent LAPORTE <laurent.laporte.pro@gmail.com>

MartinBelthle committed:
fix(table-mode): fixes the reading of UI information in the case where the study has only one area  (#1674)

Co-authored-by: Laurent LAPORTE <laurent.laporte.pro@gmail.com>

laurent-laporte-pro committed:
fix(table-mode): issue to read area information in the case where the study has only one area (#1690)

laurent-laporte-pro committed:
test: enhanced integration tests: refactored fixtures and resources

Revamped integration tests by refactoring fixtures and resources used for initializing the application during integration tests.

laurent-laporte-pro committed:
fix(variant): fixed time series deletion of renewable clusters (#1693)

laurent-laporte-pro committed:
fix(launcher): set the default number of cores to 22 (instead of 12) (#1695)

laurent-laporte-pro committed:
fix(launcher): correct endpoint for retrieving solver version list (#1672)

skamril committed:
feat(ui): remove patch number in version list when it's equal to 0 (#1698)

laurent-laporte-pro committed:
build: new bugfix release v2.14.5 (2023-08-11)

skamril commented on docs/CHANGELOG.md:
```diff
@@ -1,6 +1,13 @@
+--------------------
+
+### Bug Fixes
+
```
It's empty
